iosApp/iosApp/Pages/Onboarding/OnboardingScreenView.swift

```diff
@@ -165,21 +168,25 @@ struct OnboardingScreenView: View {
                         .accessibilityHeading(.h1)
                         .accessibilityAddTraits(.isHeader)
                         .accessibilityFocused($focusHeader, equals: .location)
+
                     Text("We use your location to show you nearby transit options.")
                         .font(Typography.title3)
-                        .padding(.bottom, 16)
+                        .padding(.bottom, 8)
                     if typeSize >= .xxxLarge, typeSize < .accessibility3 {
                         Spacer()
                     }
-                    Button(action: { shareLocation(true) }) {
-                        Text("Allow Location Services").onboardingKeyButton()
-                    }
-                    Button(action: { shareLocation(false) }) {
-                        Text(
-                            "Skip for now",
-                            comment: "Button text for deferring the request for location services"
-                        ).onboardingSecondaryButton()
+                    Button(action: {
+                        // short circuit for OnboardingPageView integration testing
+                        if skipLocationDialogue {
```

Comment:
Non-blocking nitpick: I'd maybe move this into `shareLocation`, just for cleanliness.